src/shared/utils/diagnose.ts

```diff
@@ -97,6 +100,10 @@ export const diagnose = ({
     );
   }
 
+  if (kind === "raw") {
```

Comment:
I would move this to diagnosev2 and v3
- there is a remaining mistake, the first verification of diagnose v3 should be run (check version value)

Reply:
document.version doesn't exist for unwrapped document, it will trigger an error instead

Reply:
oh yeah true